pkg/server/container_status.go

```diff
@@ -52,14 +53,17 @@ func (c *criContainerdService) ContainerStatus(ctx context.Context, r *runtime.C
 		// Based on the CRI definition, this field will be consumed by user.
 		imageRef = image.RepoDigests[0]
 	}
+	status, info := toCRIContainerStatus(ctx, container, spec, imageRef, r.GetVerbose())
```

Comment:
I prefer adding another function to generate the info.

Reply:
meh I like it this way since info is a part of the status... :-)

Reply:
Actually, it's not. It's part of the response, but not part of status. :)

Reply:
It's not part of `runtime.ContainerStatus`. It's only part of `runtime.ContainerStatusResponse`.
The helper function is used to convert information into `runtime.ContainerStatus`. So we should have a separate function for info. It's also cleaner and easier to maintain.

Reply:
You're stubborn today :-) I'll split it out and write new helper test cases for the new helper to generate the verbose part of the status response. Will be doing some cut and pasting from the other helper.

Reply:
Done.